openapi_extensions.md

```diff
@@ -41,3 +41,8 @@ same use as `format`, but that applies to the pattern of the property. We also
 define custom values for formats with the `mx-` prefix in
 `data/custom-formats.yaml`. Those values are recognized in the rendered
 specification and link to the definition of the format.
+
+## Custom `x-weight` key
```

Comment:
Unless `x-weight` is a colloquial identifier, `x-sort-weight` may be slightly more self-descriptive?

Reply:
I don't feel strongly but I thought it might be good to match the [`weight`](https://gohugo.io/methods/page/weight/) frontmatter property on pages?

Reply:
Ah, OK, I can rally behind that! `x-weight` sounds good.